src/lib/gadgets/elliptic-curve.ts

```diff
@@ -416,10 +416,10 @@ function multiScalarMul(
 
   // pack points to make array access more efficient
```

Comment:
it's a hashed point now! 
`// hashed points to make array access more efficient`